Translation revisions (#560)
07th-mod/higurashi-console-arcs: Update/koto_005.txt

//陸rvS16/46/DS43020306.本殿のご神体の中に秘蔵しておいた、巫女服（説明省略）を羽入に手渡した。
	PlaySE(4, "ps3/s16/46/ds43020306", 256, 64);
	OutputLine(NULL, "本殿のご神体の中に秘蔵しておいた、巫女服（説明省略）を羽入に手渡した。",

> 			NULL, "I presented Hanyuu with a miko outfit (explanation redacted) that I'd been keeping in the main shrine's worship statue.", Line_Normal);

Reviewer: ```suggestion
			NULL, "I presented Hanyuu with a miko outfit (explanation redacted) that I'd been keeping in the main shrine's holy statue.", Line_Normal);
```

//rそう思って、夜に寝床を抜け出すのだが、…どうしてもあと少し、あと少しだけと決心が鈍って結局、戻ってくる。…その繰り返しだ。
	OutputLine(NULL, "そう思って、夜に寝床を抜け出すのだが、…どうしてもあと少し、あと少しだけと決心が鈍って結局、戻ってくる。…その繰り返しだ。",

> 			NULL, "It was with that thought I'd try to sneak out at night... but I couldn't help but want to stay here just a little longer and my resolve would waver every time, so I'd go back... This had happened too many times than I could count.", Line_Normal);

Reviewer: ```suggestion
			NULL, "It was with that thought I'd try to sneak out at night... but every time, I couldn't help but want to stay here just a little longer and my resolve would waver, so I'd go back... This had happened too many times to even count.", Line_Normal);
```

//rとりあえず、それは彼の人柄と普段の言動からある程度の納得でもって受け入れられているようだが、さすがにあまり人前に出ないのは、変な勘繰りを生みかねない。
	OutputLine(NULL, "とりあえず、それは彼の人柄と普段の言動からある程度の納得でもって受け入れられているようだが、さすがにあまり人前に出ないのは、変な勘繰りを生みかねない。",

> 			NULL, "The villagers appeared to have accepted that story to a certain extent based on Riku's established character and demeanor, but as expected, it was easy for undue suspicions to bloom since I never showed my face.", Line_Normal);

Reviewer: ```suggestion
			NULL, "The villagers appeared to have accepted that story to a certain extent based on Riku's established character and demeanor, but as expected, it was easy for undue suspicion to bloom since I never showed my face.", Line_Normal);
```
